generate/report.go

```diff
@@ -147,6 +153,20 @@ func (r Report) empty() bool {
 		len(r.Successes) == 0
 }
 
+func (r *Report) sort() {
```

Comment:
now we manipulate the report in ways that is easy to generate it with random order (using maps). Other changes could also change the order, so it seems safer to just always sort the report before returning on code generation.